Monytor.Domain/ParsingExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Globalization;
+
+namespace Monytor.Domain {
+    public static class ParsingExtensions {
+        public static CultureInfo ParsingCulture = CultureInfo.GetCultureInfo("en-US");
```

Comment:
Not necessary: Use CultureInfo.InvariantCulture instead

Reply:
Im not sure how good timespanes are handled by the default binder in the WebApi. We can change it any time later.